files/js-cookie/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "js-cookie",
+  "repo": "js-cookie/js-cookie",
+  "files": {
+    "include": ["src/js.cookie.js"]
```

Comment:
Or if you dont want you can manually add the minified files in this PR. But then you will have to remove update.json because the bot wont be able to auto-update it.